docker/mysql-to-postgres/bin/migrate-mysql-to-postgres

```diff
@@ -125,6 +125,18 @@ if pgloader_status != 0
   exit 1
 end
 
+puts "Renaming into public schema"
+mysql_db_name = mysql_url.path.sub("/", "")
+rename_schema_cmd = "PGPASSWORD=#{db_password} psql -o /dev/stdout -U #{db_user} -h #{db_host} -p #{db_port} -d #{db_name} -c \"DROP schema public; ALTER schema #{mysql_db_name} RENAME to public;\""
```

Comment:
What happens if the MySQL database so happens to be called `public` I wonder? Will pgloader still work? Assuming it does I would reckon the data would then actually be correctly in the public schema, right? So maybe we should not do this step in that case.